### PR TITLE
Allow for whole day meetings

### PR DESCRIPTION
**Pull request by zianc**

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -112,7 +112,7 @@
         allAttendees.addAll(optionalAttendees);
         
         long duration = request.getDuration();
-        if (duration < 0 || duration >= TimeRange.END_OF_DAY) {
+        if (duration < 0 || duration > TimeRange.END_OF_DAY) {
```
Comment (yinggao) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:115]: Could you also add a test case where duration is whole day?

**Review by yinggao — CHANGES_REQUESTED** [review]

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java (hunk 1 above)
Comment (zianc) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java]: Done! Also added test case for negative duration as well to test same line.

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 2]
```diff
@@ -49,12 +49,14 @@
...
   private static final int TIME_0430PM = TimeRange.getTimeInMinutes(16, 30);
  
+  private static final int DURATION_NEGATIVE = -1;
   private static final int DURATION_15_MINUTES = 15;
   private static final int DURATION_30_MINUTES = 30;
   private static final int DURATION_60_MINUTES = 60;
   private static final int DURATION_90_MINUTES = 90;
   private static final int DURATION_1_HOUR = 60;
   private static final int DURATION_2_HOUR = 120;
+  private static final int DURATION_WHOLE_DAY = 1440;
```
Comment (yinggao) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:59]: Usually when writing numbers, prefer to write as `24 * 60` instead of `1440`, which explains where the 1440 comes from. Another way is to add a comment for 1440.

**Review by yinggao — CHANGES_REQUESTED** [review]

**Comment by zianc** [comment]
@yinggao will you approve this very quickly? Thank you! :)

**Review by yinggao — APPROVED** [review]
Sorry I missed this email thread.
